vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -416,6 +425,37 @@ private void unstashWorkspace(config, prefix) {
     }
 }
 
+private List getInitContainerList(config){
+    def initContainerSpecList = []
+    if (config.initContainerImage && config.containerMountPath) {
+        // regex [\W_] matches any non-word character equivalent to [^a-zA-Z0-9_]
+        def initContainerName = config.initContainerImage.replaceAll(/[\W_]/,"_" )
+        def initContainerSpec = [
+            name           : initContainerName,
+            image          : config.initContainerImage
+            ]
+        if (config.containerMountPath) {
+            initContainerSpec.volumeMounts = [[name: "volume", mountPath: config.containerMountPath]]
+        }
+        if (config.initContainerCommand == null) {
+            initContainerSpec['command'] = [
+                '/usr/bin/tail',
+                '-f',
+                '/dev/null'
+            ]
+        } else {
+            initContainerSpec['command'] = [
+                'sh',
+                '-c',
+                config.initContainerCommand
+            ]
+        }
+        echo "initContainerSpec configurations : ${initContainerSpec}"
```

Comment:
Should we really write this to the console? I believe with the verbose mode on the entire pod spec is anyway printed to console, no?